chia/wallet/cc_wallet/cc_utils.py

```diff
@@ -181,7 +208,7 @@ def check_is_cc_puzzle(puzzle: Program):
     return is_cc_mod(inner_f)
 
 
-def uncurry_cc(puzzle: Program) -> Optional[Tuple[Program, Program, Program]]:
+def uncurry_cc(puzzle: Program) -> Optional[Tuple[Program, bytes32, Program]]:
```

Comment:
Are you sure this `bytes32` is correct? My tests suggest `list(p.as_iter())` returns a list of `Program` objects.
```suggestion
def uncurry_cc(puzzle: Program) -> Optional[Tuple[Program, Program, Program]]:
```

Reply:
Whoops, yeah that's a mistake